### DISCUSSION TITLE
Use case and example projects

### DISCUSSION THREAD

**Discussion by steinhaug** [walkor/Workerman#995] [Q&A]
Are there any example projects to check out, or documentation explaining usage and implementation? I would love for some practical examples showing the typical way of using this library.

Maby this is something I missed out on when learning coding by myself... (Lynda.com courses on PHP was super interesting, a lot of concepts and methods you never get to hear about when "doing it by yourself"). Honestly, why do I need this library? Whats the benefits? What is the usecase? More specifically what is a practical usecase, where does this library shine? 

I have starred this project knowing I should look into it, however doesn't hurt to ask. I have a notion that I need this library, just need to figure out for what.

Anyone ? 

EDIT: Seems this page explained what I'm asking quite detailed
[PHP Websocket - Workerman](https://www.dbestech.com/tutorials/php-websocket-workerman)

**Comment by Tinywan** [comment]
Hi!

**Comment by ghost** [comment]
👋 Hello there, [chesslablab/chess-server](https://github.com/chesslablab/chess-server) is a chess server using Workerman.
